pyanaconda/modules/common/base/base.py

```diff
@@ -133,6 +136,26 @@ def collect_requirements(self):
         """
         return []
 
+    def set_locale(self, locale):
```

Comment:
This method should be probably implemented in the class MainModule, so the boss can use it as well.

Reply:
Done.